kie-wb-parent/kie-wb-webapp/src/main/resources/security-module-process-admin.properties

```diff
@@ -19,6 +19,7 @@ role.process-admin.permission.perspective.read.UserHomePagePerspective=true
 role.process-admin.permission.perspective.read.ProcessDefinitions=true
 role.process-admin.permission.perspective.read.ProcessInstances=true
 role.process-admin.permission.perspective.read.Tasks=true
+role.process-admin.permission.perspective.read.TaskAdmin=true
```

Comment:
Indeed only admin and process-admin should have access to this perspective. I'll check the issue